Amazon.QLDB.Driver/session/QldbSession.cs

```diff
@@ -15,84 +15,54 @@ namespace Amazon.QLDB.Driver
 {
     using System;
     using System.Net;
+    using System.Text.RegularExpressions;
     using Amazon.QLDBSession.Model;
     using Amazon.Runtime;
     using Microsoft.Extensions.Logging;
 
     /// <summary>
-    /// <para>Represents a session to a specific ledger within QLDB, allowing for execution of PartiQL statements and
-    /// retrieval of the associated results, along with control over transactions for bundling multiple executions.</para>
-    ///
-    /// <para>The execute methods provided will automatically retry themselves in the case that an unexpected recoverable error
-    /// occurs, including OCC conflicts, by starting a brand new transaction and re-executing the statement within the new
-    /// transaction.</para>
-    ///
-    /// <para>There are three methods of execution, ranging from simple to complex; the first two are recommended for their inbuilt
-    /// error handling:
-    /// <list type="bullet">
-    /// <item><description>Execute(string, Action), Execute(string, Action, List), and Execute(string, Action, params IIonValue[])
-    /// allow for a single statement to be executed within a transaction where the transaction is implicitly created
-    /// and committed, and any recoverable errors are transparently handled. Each parameter besides the statement string
-    /// have overloaded method variants where they are not necessary.</description></item>
-    /// <item><description>Execute(Action, Action) and Execute(Func, Action) allow for more complex execution sequences where
-    /// more than one execution can occur, as well as other method calls. The transaction is implicitly created and committed, and any
-    /// recoverable errors are transparently handled. The second Action parameter has overloaded variants where it is not
-    /// necessary.</description></item>
-    /// <item><description><see cref="StartTransaction"/> allows for full control over when the transaction is committed and leaves the
-    /// responsibility of OCC conflict handling up to the user. Transaction methods cannot be automatically retried, as
-    /// the state of the transaction is ambiguous in the case of an unexpected error.</description></item>
-    /// </list>
-    /// </para>
+    /// Represents a session to a specific ledger within QLDB, allowing for execution of PartiQL statements and
+    /// retrieval of the associated results, along with control over transactions for bundling multiple executions.
```

Comment:
What happened to the depth of comments here previously?

Reply:
It's not relevant as the QldbSession is not exposed and thus the docstrings are inaccessible except if the code is read directly, and those workflows are no longer followed nor accurate to what happens in the class.